Frameworks/XADMaster.framework/Versions/A/Headers/XADArchive.h

```diff
@@ -33,7 +33,7 @@ extern NSString *XADFinderFlags;
 	NSMutableDictionary *namedict;
 
 	off_t extractsize,totalsize;
-	int extractingentry;
+	NSInteger extractingentry;
```

Comment:
It doesn't look like you updated the XADMaster library itself. Editing headers of compiled libraries must be done with care.

Good news is that XADMaster (or a previous version of it) is open-source.

Reply:
Oops yeah this was a general copy and replace mostly, I will revert changes from the Frameworks folder. I see several mirrors/forks of XADMaster, which one does Simple Comic derive the framework from? Any others that I should look at with this code review? Thanks!

Reply:
Looks like 2 files need to be reverted:
1.  Frameworks/XADMaster.framework/Versions/A/Headers/CSInputBuffer.h
2. Frameworks/XADMaster.framework/Versions/A/Headers/XADArchive.h

I found this to be the official repo: https://github.com/MacPaw/XADMaster